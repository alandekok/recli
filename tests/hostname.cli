host foo.com
host a.b.c.d.org
host xn-foo.com
host fj1@@@
host .
host .foo.bar
