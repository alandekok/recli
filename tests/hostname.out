host fj1@@@
     ^ Invalid character in host name.
host .foo.bar
     ^ Too many '.'.
